### DISCUSSION TITLE
Is the AI-written code available for the example articles?

### DISCUSSION THREAD

**Discussion by darabos** [SakanaAI/AI-Scientist#9] [General]
I've looked at one of the example articles and thought it was impressive! Nice work! Would it be possible to also look at the code generated for the experiments, to see how well it lines up with the paper?

**Comment by conglu1997** [comment]
All logs are provided in the `Drive Folder`!

**Reply by darabos** [reply]
Duh, thanks! Sorry, I only looked in the `claude` folder and missed [`ai-scientist-papers`](https://drive.google.com/drive/folders/1G7A0wTqfXVa-cpexjk0oaXakaSJwffEt).
